src/server/controllers/user/verify.js

```diff
@@ -1,4 +1,4 @@
-import { User } from '../models/schema/users/userSchema.js';
+import { User } from '#schemas/users/userSchema.js';
```

Comment:
Missing user handler function in some cases

insead of `User.findOne` use function that u will create in `userHelper` 
second think is if you create folder `handlers` try to avoid different naming file inside.
f.ex 
❌ `handlers/userHelpers.js`
✅ `handlers/userHandlers.js`

Reply:
❓

Reply:
Also done that...